### ISSUE TITLE
incorporate changes to fix staircase bug

### ISSUE THREAD

**Issue by thartbm** [Intrepid-2a/Distance#3]
Also requires re-introducing some changes made to fit the experiment into a user-friendly workflow.

- make task into a function, so it can be started from elsewhere (and optionally ask for info not provided as arguments to the function)
- colors need to come from central locations: either the color calibration file, or the localizeSetup, but not be defined locally (and potentially differently) for each experiment
- use agreed upon folder structure for storing data (and not hard code any folders that would only exist on 1 PC in the world, such as someone's home directory)
- use window object, and fixation and fusion stimuli from central definitions in localizeSetup

There might be some other smaller changes to the order in which stuff happens. E.g. the window object should be close, but only after both the last message is displayed on screen and the eye-tracker object is closed, since both rely on the window object existing.

**Comment by thartbm** [comment]
some of the paths / folders include the participant ID, so only set paths after the ID is known

**Comment by thartbm** [comment]
centralize blind spot marker generation, with a small scale down for noisy fixation accurcay

**Comment by thartbm** [comment]
color changes:

- col_back should be different in Toronto (calibrated monitor might be solving this issue already)
- col_both used to be defined differently... I'll centralize this to localizeSetup both to simplify code and to allow passing it on to the eye-tracker object, and we can decide whichever definition to use later on

**Comment by thartbm** [comment]
No longer relevant.
